src/glyph_item_iter.rs

```diff
@@ -0,0 +1,95 @@
+// Copyright 2019, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use glib::translate::*;
+use pango_sys;
+use GlyphItem;
+
+glib_wrapper! {
+    #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
+    pub struct GlyphItemIter(Boxed<pango_sys::PangoGlyphItemIter>);
+
+    match fn {
+        copy => |ptr| pango_sys::pango_glyph_item_iter_copy(mut_override(ptr)),
+        free => |ptr| pango_sys::pango_glyph_item_iter_free(ptr),
+        init => |_ptr| (),
+        clear => |_ptr| (),
+        get_type => || pango_sys::pango_glyph_item_iter_get_type(),
+    }
+}
+
+impl GlyphItemIter {
+    pub fn init_end(glyph_item: &mut GlyphItem) -> Option<GlyphItemIter> {

```

Comment:
I gues this should probably be called `new_end` or so, as it's creating a new instance of `GlyphItemIter` rather than just initializing it.

Reply:
Maybe, I just don't want problem with docs.
@GuillaumeGomez, @sdroege what do you think?

Reply:
Yes and the signature of the function should look different

Reply:
@sdroege without `mut` ?

Reply:
This does not need a mutable reference is what I meant :)